src/lib.rs

```diff
@@ -164,18 +184,50 @@ fn internal_generate(mut args: GenerateArgs) -> Result<PathBuf> {
         info!("{}", style("Initializing a fresh Git repository").bold());
         vcs.initialize(&project_dir, branch, args.force_git_init)?;
     }
-
     println!(
         "{} {} {} {}",
         emoji::SPARKLE,
         style("Done!").bold().green(),
         style("New project created").bold(),
         style(&project_dir.display()).underlined()
     );
-
     Ok(project_dir)
 }
 
+fn test_expanded_template(template_dir: &PathBuf, args: &GenerateArgs) -> Result<PathBuf> {
+    println!(
+        "{} {}{}{}",
+        emoji::WRENCH,
+        style("Running \"").bold(),
+        style("cargo test"),
```

Comment:
here `cargo test` is hard coded, but below the actual command is taken from `std::env::var("CARGO_GENERATE_TEST_CMD")` with the default of `cargo test`.

IMO it would be better to first extract the command that will be used finally, and reuse this command here in the `println!`.

Reply:
we can tackle this as a follow up..